### ISSUE TITLE
Themes not applying fully

### ISSUE THREAD

**Issue by Cogsagi** [khanhas/spicetify-cli#243]
Hi there,

I've tried applying several themes, and while the color changes for the background, none of the content for Spotify itself loads. Screenshot attached:
![image](https://user-images.githubusercontent.com/41082782/88131772-c00ac980-cbab-11ea-881d-caecfee668ba.png)

**Comment by khanhas** [comment]
Enable and open devtool, navigate to Console to see there is any error

**Comment by Cogsagi** [comment]
This is what pops up:
![image](https://user-images.githubusercontent.com/41082782/88133175-686e5d00-cbaf-11ea-84a2-910594cf3ef7.png)

**Comment by khanhas** [comment]
Can you reinstall Spotify and run `spicetify backup apply`

**Comment by Cogsagi** [comment]
Same issue, have to restore to get anything appearing once I apply anything

**Comment by khanhas** [comment]
What do you mean by that?
Also can I see your config file?

**Comment by Cogsagi** [comment]
I meant when I do the backup apply, it just causes the same error I mentioned in the original post, and I have to spicetify restore to fix it.
Here's my config file, let me know if you meant a different one.
[config.txt](https://github.com/khanhas/spicetify-cli/files/4957880/config.txt)

**Comment by khanhas** [comment]
How did you install spicetify?
Not this file, where do you find this anyway? 
To get config file path, run `spicetify -c`

**Comment by Cogsagi** [comment]
Gotchya. I pulled that from the themes folder. I installed spotify using the powershell command in the installation section:
Invoke-WebRequest -UseBasicParsing "https://raw.githubusercontent.com/khanhas/spicetify-cli/master/install.ps1" | Invoke-Expression

Here's the correct config. It's an .ini in the folder, just had to switch to txt for the upload here
[config.txt](https://github.com/khanhas/spicetify-cli/files/4958114/config.txt)
